Automattic/jetpack
# Admin Page: don't show plugin cards when you can't install plugins

#### Changes proposed in this Pull Request:

* Follow-up from #21866
* This adds a new check to see if one can install plugins. This is especially important, where an admin with the manage_options cap may not have the capability to install plugins. There, only superadmins have that capability.

#### Jetpack product discussion

* Internal reference: p8oabR-LV-p2#comment-5806

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Set up a Multisite network, where you create:
    * a new site
    * a new admin, with access to that site.
* Log in as the new admin, go to the Jetpack menu in that new site, and connect the site to your WordPress.com account.
* When you get back to wp-admin and the Jetpack menu, you should not see the card to install the Boost plugin at the bottom of the page.
* If you log back in as the superadmin, you should see the card.
* If you try with a single WordPress installation, you should see the card as an admin.
